### PR TITLE
Added support to change java version for generic server

### PR DESCRIPTION
**Pull request by lstocchi**
This is related to https://github.com/redhat-developer/rsp-server-community/pull/54

### CODE REVIEW

**Comment by lstocchi** [comment]
@odockal this PR and https://github.com/redhat-developer/rsp-server-community/pull/54 should allow users to set vm.install.path for tomcat servers (issue https://github.com/redhat-developer/rsp-server-community/issues/53) . Let me know if you have any question.

P.S: I don't have any right on the community repo so i cannot even merge my PR. Do you have any?

File: framework/bundles/org.jboss.tools.rsp.server.generic/src/main/java/org/jboss/tools/rsp/server/generic/discovery/GenericVMRegistryDiscovery.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
+package org.jboss.tools.rsp.server.generic.discovery;
```
Comment (odockal) [inline framework/bundles/org.jboss.tools.rsp.server.generic/src/main/java/org/jboss/tools/rsp/server/generic/discovery/GenericVMRegistryDiscovery.java:1]: Please add a copyright header.

File: framework/bundles/org.jboss.tools.rsp.server.generic/src/main/java/org/jboss/tools/rsp/server/generic/servertype/GenericServerAttributes.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package org.jboss.tools.rsp.server.generic.servertype;
```
Comment (odockal) [inline framework/bundles/org.jboss.tools.rsp.server.generic/src/main/java/org/jboss/tools/rsp/server/generic/servertype/GenericServerAttributes.java:1]: copyright.

**Comment by odockal** [comment]
@lstocchi I believe that we need to release rsp first and update target platform in community to get proper bundles with changes there...
And nope, I do not have right in community to merge PR... :/

**Comment by lstocchi** [comment]
> @lstocchi I believe that we need to release rsp first and update target platform in community to get proper bundles with changes there...
> And nope, I do not have right in community to merge PR... :/

Yes, i thought the same thing. 
Daaaamn, i guess we need to wait Jeff comes back from PTO then (next Monday)

**Pull request merged**